modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -22,6 +24,8 @@ def type
             type
           )
           message = "No active '#{params[:type].downcase}' intent to file found."
+          ClaimsApi::Logger.log('ITF', type: params[:type].downcase,
+                                       detail: message.to_s)
```

Comment:
Looks like we're logging "No active <type> intent to file found" in cases where one is found.
Tested ICN=1012667145V762142 w/ type survivor (http://localhost:3000/services/claims/v2/veterans/1012667145V762142/intent-to-file/:type) and got the following log line:
2023-08-18 09:33:00.908824 I [10318:puma srv tp 008] Rails -- ClaimsApi :: ITF :: {"type":"survivor","detail":"No active 'survivor' intent to file found."} :: Location: /Users/matthewchristianson/development/vets-api/modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb:27